### PR TITLE
Updates to include parsing for additional SV fields in GPGSV sentences

### PR DESCRIPTION
**Pull request by bnhhughes**
Here's the additional SV parsing for the GPGSV messages.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,16 @@ build/
 *.d
 utests*
 systest
+
+# CMake and Eclipse files
```
Comment (jhaws1982) [inline .gitignore]: I think we can scrap all our changes here and just remove all those files if present.

Comment (bnhhughes) [reply .gitignore]: Agreed - I'll take those back out

Comment (bnhhughes) [reply .gitignore]: The .gitignore changes are gone now.

**Comment by bnhhughes** [comment]
I opted to reset and just remove all the .gitignore commits so now it doesn't ever add those changes and add additional commits to take them back out.

**Pull request merged**